include/grl/kuka/KukaJAVAdriver.hpp

```diff
@@ -314,12 +333,14 @@ namespace grl { namespace robot { namespace arm {
               logger_->info("re-extracted {}{}{}", movearm->goal()->position()->size(), " joint angles: ",angles);
           }
 
+          logger_->info("sending packet to KUKA iiwa: len = {}", fbbP->GetSize());
```

Comment:
this should be if(debug) or whatever the name of the debug variable is because it will print quite a lot